ibis/file/hdf/api.py

```diff
@@ -0,0 +1,5 @@
+from ibis.file.hdf.client import HDFClient
+
+
+def connect(dictionary):
```

Comment:
Needs a small docstring describing what `dictionary` should contain.

Reply:
done